_studio/mfx_lib/decode/vp8/src/mfx_vp8_dec_decode_hw.cpp

```diff
@@ -1284,11 +1284,20 @@ mfxStatus VideoDECODEVP8_HW::DecodeFrameHeader(mfxBitstream *in)
         while (++i < 2);
     }
 
+// On Android O we use old version of driver and should use special code for 1st partition size computation (for count == 8)
```

Comment:
Why you don't use old version of mediasdk then? What is this driver on Android O: open or closed source?

Reply:
On Android O  we have frozen version of Open-Source driver (~Spring-Summer 2018), and there is no plans to change source code of driver.
We have a plan to provide new features and bug fixing from latest Media SDK source code, and now we blocked only by broken vp8d, current PR fixed the problem (by recovering old source code).